src/js/videojs-dash.js

```diff
@@ -1,6 +1,270 @@
+/*! videojs-contrib-dash - v1.1.2 - 2016-03-07
+ * Copyright (c) 2016 Brightcove  */
 (function(window, videojs, dashjs) {
   'use strict';
 
+  var WhitelistPlugin = function () {
+    var globalManifest;
```

Comment:
We use one `var` declaration + commas [elsewhere in this project](https://github.com/nweber/videojs-contrib-dash/blob/1.x/src/js/videojs-dash.js#L304).